### PR TITLE
resource/alicloud_vpc_ipv6_gateway: adds new attribute resource_group_id, tags, business_status, create_time, expired_time, instance_charge_type, ipv6_gateway_id; deprecated attribute spec.

### PR DESCRIPTION
**Pull request by ChenHanZhang**
…d, Tags etc.

### CODE REVIEW

File: alicloud/resource_alicloud_vpc_ipv6_gateway.go [hunk 1]
```diff
@@ -23,31 +24,57 @@ func resourceAlicloudVpcIpv6Gateway() *schema.Resource {
...
 			},
 			"spec": {
-				Type:         schema.TypeString,
-				Optional:     true,
-				Computed:     true,
-				ValidateFunc: validation.StringInSlice([]string{"Large", "Medium", "Small"}, false),
+				Type:       schema.TypeString,
+				Optional:   true,
+				Computed:   true,
+				Deprecated: "IPv6 gateways do not distinguish between specifications. This parameter is no longer used.",
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_vpc_ipv6_gateway.go:71]: 废弃版本号要带上

Comment (ChenHanZhang) [reply alicloud/resource_alicloud_vpc_ipv6_gateway.go]: done

File: alicloud/resource_alicloud_vpc_ipv6_gateway.go [hunk 2]
```diff
@@ -59,198 +86,238 @@ func resourceAlicloudVpcIpv6Gateway() *schema.Resource {
...
+	d.Set("business_status", objectRaw["BusinessStatus"])
+	d.Set("create_time", objectRaw["CreationTime"])
+	d.Set("description", objectRaw["Description"])
+	d.Set("expired_time", objectRaw["ExpiredTime"])
+	d.Set("instance_charge_type", convertVpcInstanceChargeTypeResponse(objectRaw["InstanceChargeType"]))
+	d.Set("ipv6_gateway_name", objectRaw["Name"])
+	d.Set("resource_group_id", objectRaw["ResourceGroupId"])
+	d.Set("status", objectRaw["Status"])
+	d.Set("vpc_id", objectRaw["VpcId"])
+	d.Set("ipv6_gateway_id", objectRaw["Ipv6GatewayId"])
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_vpc_ipv6_gateway.go:165]: 弃用的属性，该输出还是得输出啊

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: alicloud/resource_alicloud_vpc_ipv6_gateway.go (hunk 2 above)
Comment (ChenHanZhang) [reply alicloud/resource_alicloud_vpc_ipv6_gateway.go:165]: spec已经从api上移除了

**Review by xiaozhu36 — APPROVED** [review]

**Pull request merged**